sydent/hs_federation/verifier.py

```diff
@@ -83,9 +83,7 @@ def verifyServerSignedJson(self, signed_json, acceptable_server_names=None):
         of the signatures on it
         XXX: This contains a fairly noddy version of the home server
         SRV lookup and signature verification. It only looks at
-        the first SRV result. It does no caching (just fetches the
```

Comment:
I think most of this comment is still true?

(Except, ironically, the bit you've left in place about only checking the first SRV result)

Reply:
Ah, was under the impression the agent came with built in caching. And should we leave in info about perspectives if we're no longer considering it a part of the ecosystem?

Reply:
yeah the bit about perspectives is arguably spurious right now, but equally we aren't doing any other checks on the cert (for now) so :man_shrugging: .

And no, there's no caching in the agent afaik.